### PR TITLE
Added WINNF.FT.S.ESM.1

### PR DESCRIPTION
**Pull request by sushantparkhi**

### CODE REVIEW

File: src/harness/testcases/testdata/esc_sensor_record_0.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "id": "esc_sensor/admin_0/esc_sensor_0",
+ "installationParam":{
```
Comment (kate-harrison) [inline src/harness/testcases/testdata/esc_sensor_record_0.json]: Whitespace inconsistency?

File: src/harness/testcases/testdata/esc_sensor_record_0.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+   "latitude": 39.0119,
+    "longitude": -98.4842,
+    "height": 9.3,
+    "heightType": "AGL",
+    "indoorDeployment": false,
```
Comment (kate-harrison) [inline src/harness/testcases/testdata/esc_sensor_record_0.json]: Will this field be used during protection of an ESC sensor? If not, maybe it's not required (which might unfortunately make schema validation difficult...).

Comment (sushantparkhi) [inline src/harness/testcases/testdata/esc_sensor_record_0.json]: R2-ESC-07: ESC Interference Protection:  An ESC operator may request interference protection for one or more of its ESC sensors. An ESC operator that requests protection shall provide to its affiliated SAS(s) the location and height of the protected sensor antenna, as well as the sensor system’s maximum antenna gain in all horizontal directions, specified at 1 degree increments referenced to true north. [Reference R2-SGN-08 and R2-SGN-09].

Based on R2-ESC-07: I think the required fields will be latitude, longitude, height, heightType and antennaGain(which is actually an array with 1-degree increments)

Comment (sushantparkhi) [inline src/harness/testcases/testdata/esc_sensor_record_0.json]: Probably, I can define a separate schema for ESC Sensor Data installationParam?

File: schema/InstallationParamEsc.schema.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  "required": [
+    "latitude",
+    "longitude",
+    "height",
+    "heightType",
+    "antennaGain"
+  ],
+  "additionalProperties": false,
+  "properties": {
+    "latitude": {
```
Comment (kate-harrison) [inline schema/InstallationParamEsc.schema.json]: Is there any way to reference the "parent" schema properties?

File: src/harness/testcases/testdata/esc_sensor_record_0.json [hunk 4]
```diff
@@ -1,12 +1,10 @@
...
+  "installationParam": {
+    "latitude": 39.0119,
     "longitude": -98.4842,
     "height": 9.3,
     "heightType": "AGL",
-    "indoorDeployment": false,
-    "antennaGain": 16,
-    "antennaBeamwidth": 30
- }
+    "antennaGain": [9,10,9,8,10,16,7,8,10,9,8,7,10,16,7,8,9,10,16,8,9,10,9,8,7,9,8,10,16,12,12,16,19,10,9,7,9,14,9,10,9,8,10,16,7,8,10,9,8,7,10,16,7,8,9,10,16,8,9,10,9,8,7,9,8,10,16,12,12,16,19,10,9,7,9,14,9,10,9,8,10,16,7,8,10,9,8,7,10,16,7,8,9,10,16,8,9,10,9,8,7,9,8,10,16,12,12,16,19,10,9,7,9,14,9,10,9,8,10,16,7,8,10,9,8,7,10,16,7,8,9,10,16,8,9,10,9,8,7,9,8,10,16,12,12,16,19,10,9,7,9,14,9,10,9,8,10,16,7,8,10,9,8,7,10,16,7,8,9,10,16,8,9,10,9,8,7,9,8,10,16,12,12,16,19,10,9,7,9,14,9,10,9,8,10,16,7,8,10,9,8,7,10,16,7,8,9,10,16,8,9,10,9,8,7,9,8,10,16,12,12,16,19,10,9,7,9,14,9,10,9,8,10,16,7,8,10,9,8,7,10,16,7,8,9,10,16,8,9,10,9,8,7,9,8,10,16,12,12,16,19,10,9,7,9,1,9,10,9,8,10,16,7,8,10,9,8,7,10,16,7,8,9,10,16,8,9,10,9,8,7,9,8,10,16,12,12,16,19,10,9,7,9,1,9,10,9,8,10,16,7,8,10,9,8,7,10,16,7,8,9,10,16,8,9,10,9,8,7,9,8,10,16,12,12,16,19,10,9,7,9,1,7,9,10,12,9,7,6,9,10,16,12,10,14,14,10,12,10,7]
```
Comment (kate-harrison) [inline src/harness/testcases/testdata/esc_sensor_record_0.json]: Hmm, does this mean that we need to change the SAS-SAS TS? It refers to the SAS-CBSD TS, which says this field should be a scalar. Can you start a thread with (at least) Greg Billock and myself?

File: src/harness/testcases/testdata/esc_sensor_record_0.json (hunk 2 above)
Comment (kate-harrison) [inline src/harness/testcases/testdata/esc_sensor_record_0.json]: Defining a separate schema will be annoying to maintain but I don't see any better way. Thanks for doing this.

**Pull request merged**